From: "Rafał Mużyło" <galtgendo@o2.pl>
Subject: [PATCH] win32u: fix another font->base_font regression
Message-Id: <20201114153442.GA3329@blackspire3>
Date: Sat, 14 Nov 2020 16:34:42 +0100

This regression has pretty much the same reason as the other recent one.
Simply reverting to the properly working code.

Signed-off-by: Rafał Mużyło <galtgendo@o2.pl>

diff --git a/dlls/win32u/font.c b/dlls/win32u/font.c
index 11111111111..11111111111 100644
--- a/dlls/win32u/font.c
+++ b/dlls/win32u/font.c
@@ -3410,6 +3410,7 @@ static UINT get_glyph_index_linked( struct gdi_font **font, UINT glyph )
     struct gdi_font *child;
     UINT res;
 
+    if ((*font)->base_font) *font = (*font)->base_font;
     if ((res = get_glyph_index( *font, glyph ))) return res;
     if (glyph < 32) return 0;  /* don't check linked fonts for control characters */
 

